index.bs

```diff
@@ -403,107 +410,100 @@ these steps:
 
 </div>
 
-## The {{FileSystemWriter}} interface ## {#api-filesystemwriter}
+## The {{FileSystemWritableFileStream}} interface ## {#api-filesystemwritablefilestream}
 
 <xmp class=idl>
 [Exposed=(Window,Worker), SecureContext]
-interface FileSystemWriter {
+interface FileSystemWritableFileStream : WritableStream {
```

Comment:
If you're changing the return value of getWriter(), you need to overwrite it.

Reply:
getWriter() will now return an instance of `WritableStreamDefaultWriter`